build/release/containers.sh

```diff
@@ -52,16 +52,60 @@ build() {
 
     [[ ${os} == "linux" ]] || return 0
 
-    tmpdir=$(mktemp -d)
-    trap "rm -fr $tmpdir" EXIT
-    cat <<EOF > $tmpdir/Dockerfile
+    echo "Building the rookd container"
+    tmpdir1=$(mktemp -d)
+    trap "rm -fr $tmpdir1" EXIT
+    cat <<EOF > $tmpdir1/Dockerfile
 FROM alpine:3.5
 RUN apk add --no-cache gptfdisk util-linux coreutils e2fsprogs
 COPY root /
 ENTRYPOINT ["/usr/bin/rookd"]
 EOF
-    build_artifact $os $arch $tmpdir rook/rookd rookd
-    rm -fr $tmpdir
+    build_artifact $os $arch $tmpdir1 rook/rookd rookd
+    rm -fr $tmpdir1
+
+    echo "Building the rook-client container"
+    tmpdir2=$(mktemp -d)
+    trap "rm -fr $tmpdir2" EXIT
+    cat <<EOF > $tmpdir2/Dockerfile
+FROM ubuntu
```

Comment:
the walkthrough right now assumes extra tools are in the container with the client, but that's not right. The walkthrough should really use the toolbox for anything besides the client. I'll look at updating that so we can simplify the client container